spring-cloud-kubernetes-dependencies/pom.xml

```diff
@@ -246,45 +245,6 @@
 				<scope>test</scope>
 			</dependency>
 
-			<dependency>
-				<groupId>io.fabric8</groupId>
-				<artifactId>mockwebserver</artifactId>
-				<version>${mockwebserver.version}</version>
-				<scope>test</scope>
-				<exclusions>
-					<exclusion>
-						<groupId>com.fasterxml.jackson.core</groupId>
-						<artifactId>jackson-databind</artifactId>
-					</exclusion>
-					<exclusion>
-						<groupId>com.fasterxml.jackson.core</groupId>
-						<artifactId>jackson-core</artifactId>
-					</exclusion>
-					<exclusion>
-						<artifactId>tools</artifactId>
-						<groupId>com.sun</groupId>
-					</exclusion>
-				</exclusions>
-			</dependency>
-
-			<dependency>
```

Comment:
this is the actual change in this PR, we should drop `kubernetes-client` explicit dependency, it is managed by bom anyway